test/images/agnhost/porter/porter.go

```diff
@@ -94,7 +120,19 @@ func serveTLSPort(port, value string) {
 	s := &http.Server{
 		Addr: "0.0.0.0:" + port,
 		Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			fmt.Fprint(w, value)
+			body := value
+			if JSONResponse {
+				w.Header().Set("Content-Type", "application/json")
+				j, err := json.Marshal(&jsonResponse{
+					Method: r.Method,
+					Body:   value})
+				if err != nil {
+					http.Error(w, "Internal Server Error", 500)
```

Comment:
Any reason not to interpolate `err` into the message here? Likewise for line 108.

Reply:
e.g.
```suggestion
					http.Error(w, fmt.Sprintf("Internal Server Error: %v", err), 500)
```